packages/react-router/test-app/cypress.config.ts

```diff
@@ -0,0 +1,21 @@
+import { defineConfig } from 'cypress';
+
+export default defineConfig({
+  video: false,
+  pageLoadTimeout: 6000000000,
```

Comment:
I know this carry forward from the existing config settings, but we should see if this can be removed.

Reply:
I tried removing it before, but it causes the swipe to go back tests to fail. This might be the same issue that we saw in the Vue test app.